### DISCUSSION TITLE
[PSD6] Esercizio Slide 10

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#151] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210017184-ce12991e-e904-456c-98e0-a148193ab4e5.png)

**Comment by Elia-Belli** [comment]
$ABC + \bar A + \bar B + \bar C = 1$
$ABC + \bar A B + \bar A \bar B + \bar B + \bar C = 1$
$B(\bar A + AC)+ \bar B + \bar C = 1$
$\bar A B + BC + \bar B + \bar C = 1$
$(\bar B + B)(\bar B + C)+ \bar C + \bar A B = 1$
$1 + \bar B + \bar A B = 1$
$1 = 1$

**Comment by Kikicima** [comment]
Non riesco a capire il passaggio dalla terz'ultima alla penultima riga, da dove arriva l'1 in somma?

**Comment by Elia-Belli** [comment]
@Kikicima 
$(\bar B + B)(\bar B + C) + \bar C + \bar A B  = 1(\bar B + C)+ \bar C + \bar A B  = (\bar B + C)+ \bar C + \bar A B $
a questo punto se tolgo le parentesi ho i termini $C + \bar C = 1$ 
e quindi $1 + \bar B + \bar A B$

**Comment by fraxb04** [comment]
corretto but a little tip: 
se applichi il teorema dell'assorbimento, ABC + !A diventa !A + BC, e così via !B + BC diventa !B + C .....
alla fine ti ritrovi con !A + !B + 1 = e per il teorema dell'elemento nullo, semplifichi tutto con 1

**Comment by FeddyLix17** [comment]
la soluzione di @Elia-Belli è corretta, propongo anche la mia in quanto **forse** più immediata
ABC + A&#x0304; + B&#x0304; + C&#x0304;  = 1
A&#x0304; + BC + B&#x0304; + C&#x0304; = 1
A&#x0304; + B&#x0304; + C + C&#x0304; = 1
A&#x0304; + B&#x0304; + 1 = 1
A&#x0304; + 1 = 1
1 = 1

**Comment by Kikicima** [comment]
Grazie!

**Comment by neoSnakex34** [comment]
se applico la legge di demorgan a (!a+!b+!c) ottengo !(abc) che essendo l'opposto del termine precedente (abc) dimostra che x + !x = 1
